rtsp.go

```diff
@@ -798,12 +825,12 @@ func H2645StartCode() []byte {
 }
 
 func (rc *rtspCamera) decodeAndStore(nalu []byte) error {
-	image, err := rc.rawDecoder.decode(nalu)
+	output, err := rc.rawDecoder.decode(nalu)
 	if err != nil {
 		return err
 	}
-	if image != nil {
-		rc.latestFrame.Store(&image)
+	if output != nil {
+		rc.latestOutput.Store(output)
```

Comment:
I don't understand the interaction between the pool, and `latestOutput`.

Reply:
I added Dan's documentation diff in a new commit, which should be elucidating

But basically the reason we went from `latestFrame image.Image` -> `latestOutput decoderOutput` is because we need the AVFrame in `rtsp.go`, specifically inside of`poolCleanupCallback` to `Put()` it back into the pool for reuse.